stories/CartSummary.stories.jsx

```diff
@@ -79,6 +79,7 @@ export const Editable = (args) => {
 
   return (
     <CartSummary
+      emptyCartMessage="Cart is Empty"
```

Comment:
@ikhoaA small detail, but could you please move this after the `{...args}` line for consistency?